src/components/AppHeader/AppHeader.css

```diff
@@ -2,6 +2,8 @@
 
 .appBar {
   background-color: var(--backgroundAltColor);
+  height: auto;
+  padding: 8px 2.4rem;
```

Comment:
I'm with @verythorough on using relative units instead of pixels. Any reason you prefer pixels, @erquhart?